src/OrchardCore.Modules/OrchardCore.Contents/Controllers/AdminController.cs

```diff
@@ -111,18 +111,23 @@ public async Task<IActionResult> List(ListContentsViewModel model, PagerParamete
                 model.Options.SelectedContentType = contentTypeId;
             }
 
+
+            IEnumerable<ContentTypeDefinition> contentTypeDefinitions = new List<ContentTypeDefinition>();
             if (!string.IsNullOrEmpty(model.Options.SelectedContentType))
```

Comment:
just make this a `List` and add to it. `IEnumerable` is no advantage here

Reply:
I have used an `IEnumarable` because `_contentDefinitionManager.ListTypeDefinitions()` returns an `IEnumerable`: it allow me to avoid a cast.

Reply:
ok fine with me.

Reply:
Thanks 😀